src/Modal/Modal.d.ts

```diff
@@ -2,10 +2,11 @@ import * as React from 'react';
 import { StandardProps, ModalManager } from '..';
 import { BackdropProps } from './Backdrop';
 import { PortalProps } from '../Portal';
+import { TransitionHandlers } from '../internal/transition';
 
 export interface ModalProps
   extends StandardProps<
-      React.HtmlHTMLAttributes<HTMLDivElement> & Partial<PortalProps>,
+      React.HtmlHTMLAttributes<HTMLDivElement> & Partial<PortalProps> & Partial<TransitionHandlers>,
```

Comment:
The properties aren't implemented on the component. Should we move it to the dependents (Drawer, Dialog, Snackbar, Popover?

Reply:
The properties used to be in ModalProps, but if the actual implementation is in the dependents I'd say it's more appropriate to move them there. Should I?

Reply:
Yes, please.

Reply:
Some dependents (Popover, Snackbar) still had the TransitionHandlers props so I left those alone. Also, the Snackbar props do not expose the Modal props, is that by design?